_inc/client/components/upgrade-notice-content/index.jsx

```diff
@@ -47,212 +40,29 @@ const UpgradeNoticeContent = moduleSettingsForm(
-			const assetCdnStatus = this.props.getModuleOverride( 'photon-cdn' );
-			const canDisplaySiteAcceleratorSettings = ( foundPhoton && foundAssetCdn ) && ( 'inactive' !== photonStatus || 'inactive' !== assetCdnStatus );
-
-			// Monitor any changes that should cause our main toggle to appear toggling.
-			let togglingSiteAccelerator;
-			// First Photon activating.
-			if ( ! this.props.getOptionValue( 'photon' ) && this.props.isSavingAnyOption( 'photon' ) ) {
-				if ( this.props.getOptionValue( 'photon-cdn' ) ) {
-					togglingSiteAccelerator = false;
-				} else {
-					togglingSiteAccelerator = true;
-				}
-			// Then Asset CDN activating.
-			} else if ( ! this.props.getOptionValue( 'photon-cdn' ) && this.props.isSavingAnyOption( 'photon-cdn' ) ) {
-				if ( this.props.getOptionValue( 'photon' ) ) {
-					togglingSiteAccelerator = false;
-				} else {
-					togglingSiteAccelerator = true;
-				}
-			// Then Photon deactivating.
-			} else if ( this.props.getOptionValue( 'photon' ) && this.props.isSavingAnyOption( 'photon' ) ) {
-				if ( this.props.getOptionValue( 'photon-cdn' ) ) {
-					togglingSiteAccelerator = false;
-				} else {
-					togglingSiteAccelerator = true;
-				}
-
-				// Is the Asset CDN being disabled as well?
-				if ( this.props.getOptionValue( 'photon-cdn' ) && this.props.isSavingAnyOption( 'photon-cdn' ) ) {
-					togglingSiteAccelerator = true;
-				}
-			// Then Asset CDN deactivating.
-			} else if ( this.props.getOptionValue( 'photon-cdn' ) && this.props.isSavingAnyOption( 'photon-cdn' ) ) {
-				if ( this.props.getOptionValue( 'photon' ) ) {
-					togglingSiteAccelerator = false;
-				} else {
-					togglingSiteAccelerator = true;
-				}
-			} else {
-				togglingSiteAccelerator = false;
-			}
-
 			return (
 				<div className="jp-upgrade-notice__content">
 					<p>
-						{ __( 'This release of Jetpack brings with it a major new free feature!' ) }
+						{ __( 'The features you rely on, adapted for the new WordPress editor.' ) }<br />
+						{ __( 'A new editor? Yes! Learn more.' ) }
 					</p>
 
 					<h2>
-						{ __( 'Speed up your site and its content' ) }
+						{ __( 'Build your Jetpack site with blocks' ) }
 					</h2>
 
 					<p>
-						{ __( 'Jetpack 6.7 introduces a new feature to help make your site faster. Our site accelerator already ' +
-							'offered speedier and optimized images served from our global Content Delivery Network. ' +
-							'Now we can also speed up your site by serving your static files (think CSS and JavaScript) from the same network.'
-						) }
-					</p>
-
-					<p>
-						{ __( 'Turn on one or both and see decreased page load speeds, ' +
-						'as well as reduced bandwidth usage—which may lead to lower hosting costs.'
+						{ __( 'Today, we are introducing the first wave of Jetpack-specific blocks built specifically ' +
+							'for the new editor experience: payment buttons, contact forms, maps, and markdown.'
```

Comment:
Should some of those things be capitalized?

Reply:
I think you're right! Let's capitalize "Simple Payments button, Contact form, Map, and Markdown."

Reply:
Added capitalizations in https://github.com/Automattic/jetpack/pull/10672/commits/6b5795fe4c16da4312204ee5e341c6659e4f84dd